apps/fyle/tasks.py

```
@@ -56,13 +58,15 @@ def get_filtered_expenses(workspace: int, expense_objects: list, expense_filters
     return filtered_expenses
 
 
-def import_expenses(workspace_id, accounting_export: AccountingExport = None, source_account_type: str = None, fund_source_key: str = None, is_state_change_event: bool = False, report_state: str = None, imported_from: ExpenseImportSourceEnum = None, accounting_export_id: int = None, report_id: str = None):
+def import_expenses(workspace_id, accounting_export: AccountingExport = None, source_account_type: str = None, fund_source_key: str = None, is_state_change_event: bool = False, report_state: str = None, imported_from: ExpenseImportSourceEnum = None, accounting_export_id: int = None, report_id: str = None, trigger_export: bool = False, triggered_by: ExpenseImportSourceEnum = None):
```

Pylint (3.3.7)

[refactor] Too many positional arguments (11/5)

(R0917)